Feature/svm

rgadams opened the pull request:
-Fixed an issue where semicolons were effecting the columns in .csv files.
-Added functionality for run_stance_detection.py to output to .csv files.
-Added a bash script that runs autocoding_processor.py, tweet_preprocessor.py, and tweet_preprocessor.py for a specified number of iterations, then computes the mean, median, and std. dev. of the trials.

rgadams committed:
run_stance_detection.py:
Added functionality to output f-scores to .csv files.
autocoding_processor.py:
Fixed a problem where semicolons were causing extra columns to be added.

rgadams committed:
Added bash script for repeated stance detection trials. Options for multipliers of against and neutral tweets in comparison to for tweets.
results_postprocessing.py takes a csv file an appends mean, median and standard deviation.

kvlinden reviewed (changes requested):
- Modify the bash script to replace the hard-coded coding_20100101_20180304.csv name to auto*.csv. Change autocoding_processor input to coding_filepath.
- Remove author/date comments.

rgadams committed:
Fixed issue where results were replacing .csv file and not appending.
Removed hardcoding destination for autocoding_processor.py
Removed Author/Date documentation
Fixed --noprofile trigger in bash script

rgadams committed:
Added filtering for company tweets in building trainset.

rgadams committed:
Fixed bash script not accounting for company tweet option.
